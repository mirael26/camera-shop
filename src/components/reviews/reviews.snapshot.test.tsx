import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import { reviewsMock } from '../../test/mocks';
import Reviews from './reviews';

jest.mock('./review-card/review-card', () => 'ReviewCard');

test('Reviews renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Reviews/>, {
    initialState: {
      data: { reviews: reviewsMock }
    }
  });
  expect(asFragment()).toMatchSnapshot();
});
