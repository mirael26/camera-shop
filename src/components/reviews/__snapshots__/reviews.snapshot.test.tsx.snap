// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews renders correctly 1`] = `
<DocumentFragment>
  <div
    class="page-content__section"
  >
    <section
      class="review-block"
      data-testid="reviews"
    >
      <div
        class="container"
      >
        <div
          class="page-content__headed"
        >
          <h2
            class="title title--h3"
          >
            Отзывы
          </h2>
          <button
            class="btn"
            type="button"
          >
            Оставить свой отзыв
          </button>
        </div>
        <ul
          class="review-block__list"
        >
          <reviewcard
            review="[object Object]"
          />
          <reviewcard
            review="[object Object]"
          />
        </ul>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
